src/common/gui/CSurgeSlider.h

```diff
@@ -16,6 +16,15 @@ enum CControlEnum_turbodeluxe
    kNoPopup = 1 << 22,
 };
 
+typedef enum MoveRateState
```

Comment:
I think this should be a member of CSurgeSlider not a standalone (which will mean you change the constants to CSurgeSlider::kUnInitialized and so on). My thinking is "it only makes sense in the context of the slider.

Alternately it could be a standalone named "SurgeSliderMoveRateState" I suppose but a member would be better if you don't mind!

Reply:
I agree. I hadn't thought of that tbh.